pkg/cable/status.go

```diff
@@ -0,0 +1,21 @@
+package cable
+
+type ConnectionStatus string
+
+const (
+	Established     ConnectionStatus = "ESTABLISHED"
+	Connecting      ConnectionStatus = "CONNECTING"
+	ConnectionError ConnectionStatus = "ERROR"
+	Unreachable     ConnectionStatus = "UNREACHABLE"
+	AuthError       ConnectionStatus = "AUTH ERROR"
+)
+
+type Connection struct {
+	CableName     string           `json:"cableName"`
+	ClusterID     string           `json:"clusterID"`
+	Status        ConnectionStatus `json:"status"`
+	RemoteIP      string           `json:"remoteIP"`
+	LocalIP       string           `json:"localIP"`
+	CableDriver   string           `json:"cableDriver"`
+	RemoteSubnets []string         `json:"remoteSubnets"`
+}
```

Comment:
Beyond status, do we expect these attributes to be different from the endpoint spec?

Reply:
No In fact, we could add the endpoint spec struct as an attribute itself and probably make things simpler.

The idea is that we want to expose this structure in an API, and then the operator can look at the API of it's own active gateway engine and report the list of connections as a status field

Reply:
1. Do we need to do anything if the data does not match the endpoint spec?
2. Most fields, even if stored in the driver, are just a copy of the endpoint -- so mismatch is just an error. Only IPs, and subnets are part of the peer information that actually comes from the wg device. 
3. Going forward we should probably encode the publickey in cable name

Reply:
I don't think the pk would fit in the cable name, but we could include it in the backend config map.